manifests/revoke.pp

```diff
@@ -58,7 +58,7 @@
   Openvpn::Revoke[$name]
 
   exec { "revoke certificate for ${name} in context of ${server}":
-    command  => ". ./vars && ./revoke-full ${name} ; test $? -eq 2 && touch revoked/${name}",
+    command  => ". ./vars && ./revoke-full ${name}; echo \"exit $?\" | grep -qE '(error 23|exit (0|2))' && touch revoked/${name}",
```

Comment:
This part you might want to skip because the problem arose only when working with OpenSSL 1.0.1-4 - available in Ubuntu Precise: This version doesn't exit with '2' but shows 'error 23' exactly as newer versions would do.
This problem most likely affects only a very limited set of installations, hence you could surely omit this change.
PS. As I read this now, it's not even correct.